datafusion/core/tests/sql/window.rs

```diff
@@ -1515,13 +1515,14 @@ async fn test_remove_unnecessary_sort_in_sub_query() -> Result<()> {
             "    CoalescePartitionsExec",
             "      AggregateExec: mode=Partial, gby=[], aggr=[COUNT(UInt8(1))]",
             "        RepartitionExec: partitioning=RoundRobinBatch(8), input_partitions=8",
-            "          AggregateExec: mode=FinalPartitioned, gby=[c1@0 as c1], aggr=[COUNT(UInt8(1))]",
+            "          AggregateExec: mode=FinalPartitioned, gby=[c1@0 as c1], aggr=[]",
```

Comment:
I don't understand this change -- why is there no aggregate anymore?

Reply:
Has resolved it , thanks @alamb